### PR TITLE
Entity analysis section

### PR DESCRIPTION
**Pull request by jeinygroove**
@BudiGabe It wasn't a good idea to write Charts as we did in portfolio projects, because JS implementation of Charts API manipulates with DOM elements directly. It's bad, because in React you shouldn't do it, it badly affects life-cycle of the application. 

As the result, we can receive this exception: 
`DOMException: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.`

I didn't knew that and just noticed it now, that's why I haven't insisted to use `react-google-chart` instead.

### CODE REVIEW

**Comment by jeinygroove** [comment]
I will rewrite charts to `Chart` component in this pr soon.

**Comment by RazvanBudacaTechquarter** [comment]
> I will rewrite charts to `Chart` component in this pr soon.

Hmm I didn't expect that. But it shouldn't take too much to rewrite it.

**Comment by jeinygroove** [comment]
Changed my mind, I will do an issue for this and somebody can fix it later.

File: src/main/webapp/src/components/entityAnalysisInfo.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+    }
+
+    if (this.state.entityAnalysisInfo.length == 0) {
+      return <p>No entities was found.</p>;
+    }
+
+    /**
+     * Function will draw a pie chart with static data representing the most important words in
+     * our song's context.
+     * TODO fetch data from EntityServlet.
```
Comment (RazvanBudacaTechquarter) [inline src/main/webapp/src/components/entityAnalysisInfo.js]: You can remove the TODO now.

File: src/main/webapp/src/components/entityAnalysisInfo.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    if (this.state.isLoading) {
+      return (
+        <div>
+          <CircularProgress style={{ color: "black" }} />;
+        </div>
+      );
+    }
+
+    if (this.state.entityAnalysisInfo.length == 0) {
+      return <p>No entities was found.</p>;
```
Comment (RazvanBudacaTechquarter) [inline src/main/webapp/src/components/entityAnalysisInfo.js]: This should be rephrased.
Maybe
`No entities were found` or just `No entities found`

**Review by RazvanBudacaTechquarter — COMMENTED** [review]
Seems very similar to the sentiment one.

**Review by RazvanBudacaTechquarter — APPROVED** [review]

**Review by amaurymedeiros — COMMENTED** [review]
LGTM

**Review by jgaillard85 — APPROVED** [review]

**Pull request merged**